website/docs/reference/function-properties.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Function properties
+---
+
+Function properties can be declared in `.yml` files under a `functions` key.
+
+We recommend that you put them in the `functions/` directory. You can name these files `schema.yml` or `whatever_you_want.yml`, and nest them arbitrarily deeply in subfolders within that directory.
+
+<File name='functions/<filename>.yml'>
+
+```yml
+version: 2
+
+functions:
+  - name: <string> # required
+    [description](/reference/resource-properties/description): <markdown_string> # optional
```

Comment:
this is a general property - it's not specific to functions 

https://docs.getdbt.com/reference/resource-properties/description